spec/menu_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './lib/menu'
+
+describe Menu do
+
+  it 'read menu' do
+    menu = {fish_burger: 2.0,
+            beef_burger: 4.0,
+            chicken_burger: 4.0,
+            vege_burger: 2.5}
+
+    expect(subject.read_menu).to eq menu.each { |item, price| "#{item}: £#{price}, " }
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)